test/json-decode.test.ts

```diff
@@ -397,7 +400,7 @@ describe('tuple', () => {
     });
 
     it('properly displays nested errors', () => {
-      const decoder = tuple([
+      const decoder: Decoder<[{hey: {'Howdy!': unknown}}]> = tuple([
```

Comment:
No worries, `string` does work, I originally went with `unknown` because the inference in the "not assignable" error message says it's `any`. Updated in ca16031.